<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Turn cannon:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="up">
        <property name="text">
         <string>up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="down">
        <property name="text">
         <string>down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shoot">
        <property name="text">
         <string>shoot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerUp">
        <property name="text">
         <string>power up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerDown">
        <property name="text">
         <string>power down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Choose cannonball size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="small">
        <property name="text">
         <string>Small</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="medium">
        <property name="text">
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="large">
        <property name="text">
         <string>Large</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Power:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarSpeed">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
